### PR TITLE
Create DeepShah

### PR DESCRIPTION
**Pull request by Databoy555**

### CODE REVIEW

**Comment by amblina** [comment]
Hi @Databoy555 could you confirm which platform you generated your keys on?

**Comment by Databoy555** [comment]
Windows

File: keys/DeepShah.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name:Deep Rohit Lalji Shah
```
Comment (graingert) [inline keys/DeepShah.yml]: you need a space:

```yaml
name: Deep...
fingerprint: 27...
```

Comment (graingert) [reply keys/DeepShah.yml]: also your fingerprint is invalid, it's 39chars, but should be 40

**Comment by graingert** [comment]
nice one @Databoy555

**Pull request merged**